cmake/onnxruntime_training.cmake

```diff
@@ -41,7 +41,25 @@ file(GLOB_RECURSE onnxruntime_training_runner_srcs
     "${ORTTRAINING_SOURCE_DIR}/models/runner/*.h"
     "${ORTTRAINING_SOURCE_DIR}/models/runner/*.cc"
 )
-add_library(onnxruntime_training_runner ${onnxruntime_training_runner_srcs})
+
+# perf test utils
+set(onnxruntime_perf_test_src_dir ${TEST_SRC_DIR}/perftest)
```

Comment:
can we refactor these perftest utils into a separate library? then reuse it here and in onnxruntime_perf_test

Reply:
on second thought, it's just a few files - so maybe not worth putting them in a separate library for now. if including the files directly, can we list them explicitly instead of using the glob pattern?